.github/workflows/rust_ci.yml

```
@@ -102,12 +104,13 @@ jobs:
         with:
           shared-key: zkp
 
-      # Build & Run test & Collect Code coverage
-      - name: Run cargo-tarpaulin on ironfish-zkp
-        uses: actions-rs/tarpaulin@v0.1
-        with:
-          version: "0.22.0"
-          args: --avoid-cfg-tarpaulin --manifest-path ironfish-zkp/Cargo.toml --release -- --test-threads 1
+      # Run tests to collect code coverage
+      - name: Run cargo-tarpaulin on ironfish-rust
```

```suggestion
      - name: Run cargo-tarpaulin on ironfish-zkp
```
💀 Well I wanted to trigger a rebuild, so I guess that gives me a good reason to.
I want to turn these into a matrix but it will require a bit more tedious config fiddling and I'm about tapped out on that for the day.